tests/Nox.Types.Tests/Types/Volume/VolumeTests.cs

```diff
@@ -214,7 +227,7 @@ private static void AssertAreEquivalent(Volume expected, Volume actual)
 
     private static void AssertAreNotEquivalent(Volume expected, Volume actual)
     {
-        var a =LengthUnit.Foot;
```

Comment:
Can you please remove the two unused variables.

Reply:
Required changes were applied